requirements-dev.txt

```diff
@@ -1,6 +1,10 @@
 autopep8 # for code formatting
-pip # listed here so we can upgrade it as same time as everything else
```

Comment:
actually, wait a minute ...
why wasn't it a problem in the past, requiring pip from here?

Just curious, I can totally see theoretical issues trying to upgrade pip simultaneously with everything else, but it seemed to work before and was simpler

Reply:
I think updating the version of the builder caused the issue. [Prior to this PR](https://github.com/awslabs/aws-crt-builder/pull/224/files), AL2 would always update its version of pip and so it would never run into the issue where it would try to update pip because it always had the latest.

At least, that is my hunch based on what's changed and what could cause it.

Reply:
I just checked and yeah, that seems to be the issue: https://github.com/awslabs/aws-crt-python/actions/runs/4885817077/jobs/8720370025

Reply:
It seems like it might also be what is causing the markupsafe issue?
https://github.com/awslabs/aws-crt-python/actions/runs/4885817077/jobs/8720373430